### DISCUSSION TITLE
Add deployment status to Scripted Record page

### DISCUSSION THREAD

**Discussion by Fluzzroz** [burkybang/SuiteAdvanced-Feedback#88] [Ideas]
I really love the "Scripts" link that show up in the top right of records, which takes the user to the corresponding Scripted Record page.
What I find lacking on this page is that it does not show which script is running or not. 
The _status_ column helps, but to really know if a script is running, it needs to have an active deployment record, have an active script record, and be in released status (or testing status and belonging to the current user). 
So would it be possible to add this _running/activated & deployed_ column? 
What about going one step further and highlighting in red the scripts that are not running?

![scripted record](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/33040640/752f04d0-ef87-4a45-ae1d-0bb882753207)

credit: This one is inspired by another add-on called NetSuite Scripted Records.

**Comment by burkybang** [comment]
Thanks for the idea! This'll be in the next release.

**Comment by Fluzzroz** [comment]
wow, this is amazing! I can now get rid of another chrome addon because yours does it better!
